comet/utils/redis.py

```
@@ -0,0 +1,191 @@
+                    retry_on_timeout=True,
+                    health_check_interval=30
+                )
+            else:
+                logger.error("REDIS_URI not configured")
+                return False
+            
+            await self.redis.ping()
+            self._connected = True
+            info = await self.redis.info('server')
+            logger.success(f"🔗 Redis connection established - Server: {info.get('redis_version', 'unknown')}")
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)